connectors/sink-google-sheets/README.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Google Sheet
+---
+
+# Google Sheet Sink
+
+## Introduction
+
+The Google Sheet Sink is a [Vanus Connector][vc] which aims to send incoming CloudEvents in a JSON format to a Google Sheet.
+
+For example, if the incoming CloudEvent looks like::
+
+```json
+{
+  "id": "88767821-92c2-477d-9a6f-bfdfbed19c6a",
+  "source": "quickstart",
+  "specversion": "1.0",
+  "type": "quickstart",
+  "time": "2022-07-08T03:17:03.139Z",
+  "datacontenttype": "application/json",
+  "data": {
+    "id": "1",
+    "name": "Ehis",
+    "email": "ehis@gmail.com",
+    "description": "Developer"
+  }
+}
+```
+
+The Google Sheet Sink will extract _data_ field and write it to a Google Sheet.
+
```

Comment:
@JieDing asked me to make some modifications to it

Reply:
@ehisakhile What about everything else, quick start etc...
This readme is not correct

Reply:
@Michaelg22 
I didn't add those other parts yet because it hasn't been made to run with docker or k8s. 

I just made a summary of the Sink connector